Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_VertexData.azsli

```diff
@@ -39,8 +39,7 @@ struct VsOutput_BasePBR
     precise linear centroid 
     float4 position : SV_Position;
     float3 normal: NORMAL;
-    float3 tangent : TANGENT; 
-    float3 bitangent : BITANGENT; 
+    float4 tangent : TANGENT;
```

Comment:
If we are removing bitangent from VsOutput then we should remove it from VsInput too, I don't think it is used in that case.